src/components/InfoIcon.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <span>
+    <font-awesome-icon
```

Comment:
Suggest to have a "active state" on the icon or text if you 'force' the popup to stay open via click.
( I did not understand how to close a random popup dat stayed open after use )

Reply:
I've made them focusable and added hide on blur.

Reply:
I've made them focus and close on blur.